openfisca_us/variables/gov/states/md/tax/income/exemption/md_aged_blind_exemptions.py

```diff
@@ -0,0 +1,58 @@
+from openfisca_us.model_api import *
+
+
+class md_aged_blind_exemptions(Variable):
```

Comment:
Could you split these into two variables, one for aged and blind and the other for aged dependents? That's how it shows up in the tax [form](https://www.marylandtaxes.gov/forms/21_forms/502_502B.pdf).